src/api/spec/controllers/webui/user_controller_spec.rb

```diff
@@ -0,0 +1,184 @@
+require 'rails_helper'
+
+RSpec.describe Webui::UserController do
+  let!(:user) { create(:confirmed_user, login: "tom") }
+  let!(:non_admin_user) { create(:confirmed_user, login: "moi") }
+  let!(:admin_user) { create(:admin_user, login: "king") }
+  let(:deleted_user) { create(:deleted_user) }
+
+  describe "GET #index" do
```

Comment:
@hennevogel using   `use_before_action(:require_admin) }` only ensures that we have this before_action/filter defined in the controller... nothing to do with checking if the filter was running inside an action.
If the `before_action` have `:except` or `:only` options... then you can't check with this if the filter was running or not in your example after a http call